examples/with-emailpassword-vercel/src/App.js

```diff
@@ -10,13 +10,13 @@ import SessionExpiredPopup from "./SessionExpiredPopup";
 
 export function getApiDomain() {
     const apiPort = process.env.REACT_APP_API_PORT || 3001;
-    const apiUrl = window.location.hostname || `http://localhost:${apiPort}`;
+    const apiUrl = process.env.REACT_APP_VERCEL_URL !== undefined ? window.location.hostname  : `http://localhost:${apiPort}`;
     return apiUrl;
 }
 
 export function getWebsiteDomain() {
     const websitePort = process.env.REACT_APP_WEBSITE_PORT || 3000;
-    const websiteUrl = window.location.hostname || `http://localhost:${websitePort}`;
+    const websiteUrl = process.env.REACT_APP_VERCEL_URL !== undefined ? window.location.hostname  : `http://localhost:${websitePort}`;
```

Comment:
In the frontend, `process.env.REACT_APP_VERCEL_URL` will always be `undefined`. Just make it `const websiteUrl = window.location.hostname || `http://localhost:${websitePort}`;`